app/views/complaints/_complaint_unlink_modal.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="margin-y-3">
```

Comment:
I moved this out into its own partial. The only actual changes to this section are all the awkward places sticking in tta vs monitoring, like `<id="unlink-modal-<%= item %>"`